packages/wifi_iot/lib/wifi_iot.dart

```diff
@@ -425,7 +433,13 @@ class WiFiForIoTPlugin {
     } on MissingPluginException catch (e) {
       print("MissingPluginException : ${e.toString()}");
     }
-    return bResult ?? false;
+
```

Comment:
Include "failed" as normal choice AND as fallback.
Add curly brackets.

```
if (iResult == 0) {
  return RegisterWifiNetworkResult.success;
} else if (iResult = 1) {
  return RegisterWifiNetworkResult.failed;
} else if (iResult == 2) {
  return RegisterWifiNetworkResult.alreadyRegistered;
} else {
  return RegisterWifiNetworkResult.failed;
}
```

Reply:
Fixed